crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -1255,6 +1256,22 @@ impl CumulocityConverter {
             }
         }
     }
+
+    pub fn is_message_tedge_agent_up(&self, message: &Message) -> Result<bool, ConversionError> {
```

Comment:
Let's keep this for now. However, when `tedge-agent` will support software list over `te` topics (see https://github.com/thin-edge/thin-edge.io/pull/2235), then will have to revised.

The mapper will have then to react to the `software-list` capability messages sent by the agents running on the main device as well as child devices.